modules/eslint-plugin/src/rules/effects/no-multiple-actions-in-effects.ts

```diff
@@ -58,7 +58,7 @@ export default createRule<Options, MessageIds>({
           });
         } else if (
           type.isUnion() &&
-          type.types.some((ut) => !typeChecker.isArrayType(ut))
+          type.types.some((ut) => typeChecker.isArrayType(ut))
```

Comment:
Good catch!
Could you also add an invalid code snippet please?

Reply:
Isn't the invalid code snippet already handled by cases like this one:
```typescript
import { of } from 'rxjs'
@Injectable()
export class Effects {
  readonly effectNOK3$ = createEffect(() =>
    this.actions$.pipe(concatMapTo(condition ? [foo(), bar()] : of(foo())))
                                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  [${messageId}]
  )
}
```

Reply:
You're right! Sorry, I was only looking at the changes, not the already existing cases.